acai-ios/CollectionViewCells/MenuCollectionViewCell.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  MenuCollectionViewCell.swift
+//  acai-ios
+//
+//  Created by Jaewon Sim on 3/17/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class MenuCollectionViewCell: UICollectionViewCell {
+    
+    // MARK: View vars
+    var imageView: UIImageView!
+    
+    var titleLabel: UILabel!
+    var ingredientsLabel: UILabel!
+    var detailStackView: UIStackView! //include title and ingredients
+
+    var additionalPricingLabel: UILabel!
+    var selectButton: UIButton!
```

Comment:
prefer that these variables are alphabetized or grouped by type, up to @drewsdunne or @kevinchan159